### ISSUE TITLE
define where images are taken from

### ISSUE THREAD

**Issue by kp0296** [Geta/ImageOptimization#2]
we use appdata to store all of the images, is there somewhere we can input some path so that the package knows where to take the images from?

**Comment by frederikvig** [comment]
Do you mean ability to specify which folders that should be optimized?

**Comment by kp0296** [comment]
Yes, currently it is only optimizing a portion of the total images that I have. There are thousands which it is just completely missing so I think it is looking in the wrong path maybe

Date: Tue, 4 Aug 2015 04:53:59 -0700

**Comment by frederikvig** [comment]
What happens when you run the scheduled job multiple times? Once it has optimized a image it won't try and optimize it again so the count should reflect that.

**Comment by kp0296** [comment]
I know, but it only optimizes 327 images. It does not go through the thousands I have in my appdata/blobs/ folder which is what I find strange. I also don't know where it is getting the 327 images from as I cannot debug. I also cannot integrate your source to my solution because it is outdated and will require many updates/changes.

Date: Fri, 4 Sep 2015 09:45:41 -0700

**Comment by frederikvig** [comment]
To clairfy, everytime you run the scheduled job it says 327 images or it said 327 images optimized and then when you run the job after that it says 0 images optimized?

**Comment by kp0296** [comment]
Second one. I run it, it says 327 images optimized, run it again then it says 0 images optimized. 

Date: Wed, 9 Sep 2015 07:58:40 -0700

**Comment by frederikvig** [comment]
Could you try uploading a new image (un-optimized) and running the job again? It should find the new image and optimize it. For the old ones it could be that they're already optimized.
